### ISSUE TITLE
Remove generated files from git

### ISSUE THREAD

**Issue by matsduf** [zonemaster/zonemaster-gui#92]
The following files are stored in Git, but they are generated from the equivalent .md versions of the FAQ. They should be removed:

```
src/assets/faqs/gui-faq-da.html
src/assets/faqs/gui-faq-en.html
src/assets/faqs/gui-faq-fr.html
src/assets/faqs/gui-faq-ru.html
src/assets/faqs/gui-faq-sv.html
```

**Comment by matsduf** [comment]
@pamasse, could you look at this issue? It should be easy to fix.

**Comment by sandoche2k** [comment]
@pamasse can you look at this issue?

**Comment by matsduf** [comment]
Resolved by #97.
